__tests__/shared/didCommPacking.ts

```diff
@@ -68,6 +68,8 @@ export default (testContext: {
         packing: 'none',
         message,
       })
+      console.log("packedMessage: ", packedMessage)
```

Comment:
If these logs were used for debug, they should be removed to reduce the noise in tests